docs/decisions/index.md

```diff
@@ -0,0 +1,13 @@
+# Architectural Decision Log
+
+This log lists the architectural decisions for [project name].

```

Comment:
Should `[project name]` be updated here?

Reply:
Hmm good question, I guess for the platform I should put this as `Platform`, but for the other templates like `template-infra` and `template-application-flask` we could have something like `docs/decisions/application` and `docs/decisions/infra/`?